### PR TITLE
add set stm files function

### PR DESCRIPTION
**Pull request by christophmluscher**
adds a function to create and set the stm files in the meta system class from the bliss corpus file

### CODE REVIEW

File: meta/system.py [hunk 1]
```diff
@@ -166,6 +166,11 @@ def set_kaldi_scorer(self, corpus, mapping):
         }
         self.scorer_hyp_arg[corpus] = "ctm"
 
+    def set_stm_files(self, corpus, **kwargs):
```
Comment (JackTemaki) [inline meta/system.py:169]: we should start to add docstring to all functions.

**Review by JackTemaki — CHANGES_REQUESTED** [review]

File: meta/system.py [hunk 2]
```diff
@@ -166,6 +166,15 @@ def set_kaldi_scorer(self, corpus, mapping):
...
+        """
+        create and set the stm files
+        :param Path|str corpus: Bliss corpus file path
```
Comment (JackTemaki) [inline meta/system.py:172]: But this is wrong now or not?
This should be just the corpus "name" as string, right?

**Review by JackTemaki — CHANGES_REQUESTED** [review]

File: meta/system.py (hunk 2 above)
Comment (christophmluscher) [reply meta/system.py:172]: ah yes you are right..

File: meta/system.py [hunk 3]
```diff
@@ -166,6 +166,15 @@ def set_kaldi_scorer(self, corpus, mapping):
         }
         self.scorer_hyp_arg[corpus] = "ctm"
 
+    def set_stm_files(self, corpus, **kwargs):
```
Comment (curufinwe) [inline meta/system.py:169]: From a set function I expect to have a parameter for the value that I want to set. A better name for what this function does is `create_stm_from_corpus`.

**Review by curufinwe — CHANGES_REQUESTED** [review]

**Pull request merged**